static/templates/actions_popover_content.handlebars

```diff
@@ -8,8 +8,8 @@
   </li>
 
   <li>
-    <a href="#" class="respond_button">
-      <i class="icon-vector-share-alt"></i> {{t "Reply" }}
+    <a href="#" class="respond_button" data-msgid="{{message.id}}">
```

Comment:
let's use data-message-id as the name for this; we just replaced all the use of data-msgid with that since it's a bit more consistent with other data elements we use :).